WDAC-Policy-Wizard/app/src/ConfigTemplate_Control.cs

```diff
@@ -505,23 +505,49 @@ private bool ReadSetRules(object sender, EventArgs e)
                 {
-                    case WDAC_Policy.NewPolicyTemplate.WindowsWorks:
-                        // Windows Works Mode 
-                        xmlPathToRead = System.IO.Path.Combine(this._MainWindow.ExeFolderPath, "DefaultWindows_Audit.xml");
-                        break;
-
-                    case WDAC_Policy.NewPolicyTemplate.SignedReputable:
-                        // Signed and Reputable Mode
-                        xmlPathToRead = System.IO.Path.Combine(this._MainWindow.ExeFolderPath, "SignedReputable.xml");
-                        break;
-
-                    case WDAC_Policy.NewPolicyTemplate.AllowMicrosoft:
-                        // Allow Microsoft mode
-                        xmlPathToRead = System.IO.Path.Combine(this._MainWindow.ExeFolderPath, "AllowMicrosoft.xml");
-                        break;
+                    // Multi-policy Format Policy Templates
+                    switch (this.Policy._PolicyTemplate)
+                    {
+                        case WDAC_Policy.NewPolicyTemplate.WindowsWorks:
+                            // Windows Works Mode 
+                            xmlPathToRead = System.IO.Path.Combine(this._MainWindow.ExeFolderPath, "DefaultWindows_Audit.xml");
+                            break;
+
+                        case WDAC_Policy.NewPolicyTemplate.SignedReputable:
+                            // Signed and Reputable Mode
+                            xmlPathToRead = System.IO.Path.Combine(this._MainWindow.ExeFolderPath, "SignedReputable.xml");
+                            break;
+
+                        case WDAC_Policy.NewPolicyTemplate.AllowMicrosoft:
+                            // Allow Microsoft mode
+                            xmlPathToRead = System.IO.Path.Combine(this._MainWindow.ExeFolderPath, "AllowMicrosoft.xml");
+                            break;
+                    }
+                }
+                else
+                {
+                    // Legacy/single-policy Format Policy Templates
+                    switch (this.Policy._PolicyTemplate)
+                    {
+                        case WDAC_Policy.NewPolicyTemplate.WindowsWorks:
+                            // Windows Works Mode 
+                            xmlPathToRead = System.IO.Path.Combine(this._MainWindow.ExeFolderPath, "DefaultWindows_Audit - SingleFormat.xml");
+                            break;
+
+                        case WDAC_Policy.NewPolicyTemplate.SignedReputable:
+                            // Signed and Reputable Mode
+                            xmlPathToRead = System.IO.Path.Combine(this._MainWindow.ExeFolderPath, "SignedReputable - SingleFormat.xml");
+                            break;
+
+                        case WDAC_Policy.NewPolicyTemplate.AllowMicrosoft:
+                            // Allow Microsoft mode
+                            xmlPathToRead = System.IO.Path.Combine(this._MainWindow.ExeFolderPath, "AllowMicrosoft - SingleFormat.xml");
```

Comment:
It seems like there should be a better way than to hard-code these values as strings, here.

Reply:
You can put them in the resource.resx, or as a static internal const string in a namespace somewhere to keep them private to the assembly.

Reply:
if the goal is to make these starter templates, I'd recommend probably just compiling the XML into the binary itself as a resource.

Reply:
yeah, this is a better approach. I will just package the xml as a resource